code/html/index.html

```diff
@@ -620,6 +620,34 @@ <h2>Device administration and security settings</h2>
                                     <div class="pure-u-1 pure-u-lg-3-4 hint">Request password when starting telnet session</div>
                                 </div>
 
+                                <div class="pure-g">
+                                    <label class="pure-u-1 pure-u-lg-1-4">Heartbeat mode</label>
+                                    <select class="pure-u-1 pure-u-lg-1-4" name="hbMode" tabindex="15" >
+                                        <option value="0">HEARTBEAT_NONE</option>
```

Comment:
Use the hints here as option text, the HEARTBEAT_NONE,... keys here mean nothing for the end user. And we save some space removing the hints below.

Reply:
I agree. Will do a fix.